manifests/params.pp

```diff
@@ -4,11 +4,6 @@
   case $facts['os']['family'] {
     'Debian': {
       case $facts['os']['release']['major'] {
-        '12.04': {
-          $net_ipv4_tcp_rmem = '4096 87380 16777216'
-          $net_ipv4_tcp_wmem = '4096 65536 16777216'
-          $java_package = 'openjdk-7-jre-headless'
-        }
         '18.04': {
```

Comment:
This looks like it should be default. `openjdk-7-jre-headless` didn't exist in Stretch (oldoldstable) so unlikely to exist in anything newer. On the other hand, version 8 is only in Stretch and Buster appears to only have version 11.